clearbit/prospector.go

```diff
@@ -48,15 +62,15 @@ type ProspectorService struct {
 func newProspectorService(sling *sling.Sling) *ProspectorService {
 	return &ProspectorService{
 		baseSling: sling.New(),
-		sling:     sling.Base(prospectorBase).Path("/v1/people/"),
+		sling:     sling.Base(prospectorBase).Path("/v2/people/"),
```

Comment:
I thought all the v2 changes were going into the v1 route?

Reply:
Good catch! Yep this PR been sitting for a week